sphinxdocs/private/sphinx.bzl

```diff
@@ -177,6 +178,9 @@ def sphinx_docs(
         **common_kwargs
     )
 
+    common_kwargs_with_manual_tag = dict(common_kwargs)
+    common_kwargs_with_manual_tag["tags"] = list(common_kwargs.get("tags") or []) + ["manual"]
```

Comment:
Pigweed repo is running v0.40.0 of rules_python so I think I'm just accidentally re-introducing some code that has been deleted

Reply:
I synced this PR to head and resolved conflicts.